packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
+Conditionele velden of ‘selectie-afhankelijke vragen’ zijn formulierelementen die getoond worden als reactie op een eerdere keuze of invoer.
 
-### ‘Floating labels’
+### Visuele vormgeving
 
-A label placed in an input field can confuse a user as text is already displayed in the input field. Legibility also suffers since contrast between foreground (text) and background color is often too low. It can also be confused with inactive or placeholder text. Sometimes it might also be necessary to show actual placeholder text.
+#### Eenkoloms-layout
 
-## Placeholder text
+Om iemand comfortabel een formulier te laten scannen en een lineaire dialoog te tonen is het raadzaam om formulierelementen niet in meerdere kolommen te tonen.
 
-Optional placeholder text can assist in determining the correct data format requested for a particular input.
+#### Breedte van tekstinvoervelden
 
-A telephone number input field for example can display a placeholder explaining the desired data format (+31 010 12 345 679).
+Een verschil in breedte kan worden toegepast op tekstinvoervelden om de verwachte lengte van invoer aan te geven. Zo is een huisnummer vaak korter dan een telefoonnummer.
 
-Placeholder text must be legible and therefore have a correct contrast ratio. Because users could confuse the placeholder text with entered text, it is advisable to avoid placeholder text.
+In enkele gevallen kunnen invoervelden naast elkaar getoond worden, bijvoorbeeld het tonen van een toevoeging naast een huisnummer veld. Dit is echter alleen mogelijk als de velden een directe relatie met elkaar hebben en in een logische volgorde staan.
 
-## Helper text
+##### Voorbeelden van verschillen in breedtes van tekstinvoervelden
 
-Optional helper or feedback text is secondary to the primary action of completing a text entry. This text can be dynamic, such as when a field is filled incorrectly and you want to explain how to correct this error.
+###### IBAN nummer
 
-Helper text is shown below the entry field.
+Een IBAN nummer heeft altijd 18 posities. Dit vaste aantal posities wordt voorgelezen door screenreaders.
 
-## Validation and feedback
+###### Voorbeeld telefoonnummer
 
-A form element should not be validated until the focus of that element is unset. This makes it possible to immediately display an error message if, for example, the entered format does not meet the requirements for that field. As such, a user immediately sees the error and can fix it immediately.
+#### Verzend knop
 
-Feedback should only be shown on incorrect fields in order to minimize the mental strain on users.
+Voorkom dat een eventuele knop om het formulier te wissen geplaatst wordt in nabijheid van de knop om een formulier te versturen. Hierdoor voorkom je het risico dat een bezoeker per ongeluk een forumulier wist.
```

Comment:
Ik zou in het geval van wissen ook altijd een "destructieve actie" patroon verwachten waar een bevestiging via dialoog nodig is voordat het formulier daadwerkelijk wordt gewist.

Reply:
ik zou een inline message plaatsen als men op reset klikt. De reset button moet gepositioneerd worden op ruime afstand van de submit button.

Reply:
Een ‘destructieve actie’ patroon is inderdaad een goed idee en een patroon op zich. Ik heb issue #113 hier voor aangemaakt. T.z.t. kunnen we dan een kruisverwijzing hier tussen aanbrengen.